### PR TITLE
small revision of the intro

### PR DESCRIPTION
**Pull request by simongravelle**
I am going over the text again and fix the last remaining issue + improve the grammar when necessary.

### CODE REVIEW

File: lammps-tutorials.tex [hunk 1]
```diff
@@ -268,81 +262,87 @@ \subsection{Scope}
...
+here, as it offers features that make it particularly convenient for
+tutorials, but other console or graphical text editors, such as GNU nano,
+vi/vim, Emacs, Notepad, Gedit, and Visual Studio Code, can also be
+used.  LAMMPS can be executed either directly from
+LAMMPS--GUI (\hyperref[using-lammps-gui-label]{Appendix~\ref{using-lammps-gui-label}})
+or from the command line (\hyperref[command-line-label]{Appendix~\ref{command-line-label}}),
+the latter of which requires some familiarity with executing commands
+from a terminal or command-line prompt.
 
 In addition, prior knowledge of the theoretical basics of molecular
```
Comment (simongravelle) [inline lammps-tutorials.tex:330]: @akohlmey @jrgissing If you have other suggestion of book you like, please feel free to add it. I know there are other classic book out there, but I did not want to write the name of a book I did not open.

Comment (simongravelle) [reply lammps-tutorials.tex:330]: I am refering to the paragraph starting with "In addition, prior knowledge of the theoretical basics of molecular ... "

File: lammps-tutorials.tex [hunk 2]
```diff
@@ -231,34 +231,28 @@ \section{Introduction}
...
 interactions between atoms (the so-called force field)
-\cite{van2018validation}.  These choices are independent of the
-simulation software, though they can sometimes be limited by the availability of
-features in a given package.  The tutorials presented in this
-article aim to flatten the learning curve, and help users perform
-accurate and reliable MS using LAMMPS.
+\cite{wong2016good, van2018validation, prasad2018best}.  While these choices are independent of the
+simulation software, they may occasionally be constrained by the
+features available in a given package.  The tutorials in this
+article aim to ease the learning curve and guide users in
```
Comment (akohlmey) [inline lammps-tutorials.tex:248]:
```suggestion
article aim to flatten the learning curve and guide users in
```

Comment (akohlmey) [reply lammps-tutorials.tex:248]: Sorry, but "ease the learning curve" makes no sense. You can "ease learning" but not a curve.

File: lammps-tutorials.tex (hunk 1 above)
Comment (akohlmey) [reply lammps-tutorials.tex:330]: The main text books that I used while learning MD were the Allen/Tildesley and Hansen/McDonald books. In addition I used D. A. McQuarrie, Statistical Mechanics, Harper & Row, New York, 1976, but I think any decent text book on statistical mechanics or statistical thermodynamics will do. A good online (meta) resource would be http://www.sklogwiki.org/

File: lammps-tutorials.tex (hunk 2 above)
Comment (simongravelle) [reply lammps-tutorials.tex:248]: You are right, my bad.

**Comment by simongravelle** [comment]
I am merging that one to work on @jrgissing PR without conflict.

**Pull request merged**